enhancing/enhancer.js

```diff
@@ -71,11 +75,23 @@ function get(item) {
   if (!name || typeof(durability) !== 'number' || typeof(enhancement) !== 'number') {
     return 'error: object must include name, durability, and enhancement properties';
   }
-
-  if (enhancement > 0) name = `[+${enhancement}] ${name}`;
+  const regEx = /^\[\+\d+\]/;
```

Comment:
very nice! regex can be tricky but it's always good to at least have some of the basic matcher syntax familiarized